foods.html

```diff
@@ -39,9 +39,14 @@
           </div>
           <div class="food-table-div">
             <table class="food-table" id='food-table'>
+              <thead>
               <tr class='food-table-headers'>
                 <th class="cell-one food-tbl-name-header">Name</th><th class="food-tbl-cal-header">Calories</th>
               </tr>
+            </thead>
+            <tbody>
+              
+            </tbody>
             </table>
```

Comment:
The internet says we should have the tbody/thead tags to make it a "valid table" - it didn't break anything